src/common.rs

```diff
@@ -329,6 +329,45 @@ impl ConnectionConfig {
     }
 }
 
+pub fn get_max_allowable_transfer_amount(
+    connection_config: ConnectionConfig,
+    account_id: near_primitives::types::AccountId,
+) -> color_eyre::eyre::Result<NearBalance> {
+    let storage_amount_per_byte = actix::System::new()
+        .block_on(async {
+            near_jsonrpc_client::new_client(connection_config.rpc_url().as_str())
+                .EXPERIMENTAL_protocol_config(
+                    near_jsonrpc_primitives::types::config::RpcProtocolConfigRequest {
+                        block_reference:
+                            near_jsonrpc_primitives::types::blocks::BlockReference::Finality(
+                                near_primitives::types::Finality::Final,
+                            ),
+                    },
+                )
+                .await
+        })
+        .map_err(|err| color_eyre::Report::msg(format!("RpcError: {:?}", err)))?
+        .config_view
+        .runtime_config
+        .storage_amount_per_byte;
+    let account_items = match check_account_id(connection_config, account_id)? {
+        Some(account_view) => (
+            account_view.amount,
+            account_view.locked,
+            account_view.storage_usage,
+        ),
+        None => (0, 0, 0),
+    };
+    let (amount, locked, storage_usage) = account_items;
+    Ok(NearBalance::from_yoctonear(
```

Comment:
This function does not behave as I expected. It is called `get_max_allowable_transfer_amount` but it returns plain account balance. It means that we can run into a situation like this:
![image](https://user-images.githubusercontent.com/4030422/131985477-1c830db9-ba47-4ea0-ada9-b6c4d6f80d35.png)

Also, I believe that it's better to have a separate function for account balance or account state and then just use it in this function. What I mean (pseudocode):
```rust
func getAccState(acc): AccState {...}

func getMaxTransferAmmount(accState): NearAmount {
    return accState.amount - accState.locked;
}
```

This will allow us to reuse `getAccState` elsewhere.